ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-cpou-upgrade-from-4.15.yaml

```diff
@@ -0,0 +1,139 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  intermediate-arm64:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.15"
+  latest-arm64:
+    release:
+      architecture: arm64
+      channel: fast
+      version: "4.15"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.17"
+  target-arm64:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: baremetalds-ipi-ovn-ipv4-f14
+  cron: 38 17 13,28 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest-arm64
+    env:
+      ARCHITECTURE: arm64
+      FILTERS_ADDITIONAL: ~TELCO&
+      NVME_DEVICE: /dev/nvme1n1
+      PACKET_METRO: dc
+      PACKET_PLAN: c3.large.arm64
+      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @upgrade-check and @4.15 and @baremetal-ipi'
+      UPGRADE_PRE_RUN_TAGS: '@arm64 and @upgrade-prepare and @4.15 and @baremetal-ipi'
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_ARM64_INTERMEDIATE},${RELEASE_IMAGE_ARM64_TARGET}"
+        > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_ARM64_INTERMEDIATE
+        name: release:intermediate-arm64
+      - env: RELEASE_IMAGE_ARM64_TARGET
+        name: release:target-arm64
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: gcp-ipi-proxy-private-f14
```

Comment:
Seen from 4.15 e2e ci jobs, the job is existing in mult-arch ci, to align with it, suggest to move to mult-arch ci.

Reply:
hmm, it seems to be in 4.15 arm64/amd64 e2e ci

```
$ grep gcp-ipi-proxy-private-f14 openshift-openshift-tests-private-release-4.15*
openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.14.yaml:- as: gcp-ipi-proxy-private-f14
openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:- as: gcp-ipi-proxy-private-f14
openshift-openshift-tests-private-release-4.15__arm64-stable.yaml:- as: gcp-ipi-proxy-private-f14
```
btw, we follow the [matrix](https://issues.redhat.com/browse/OTA-1196?focusedId=25259708&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25259708) to distribute the coverage on amd64/arm64/multi payload.

Reply:
Your grep result are from stable ci. 
```
$ grep -r "as: gcp-ipi-proxy-private" *release-4.15*multi*
openshift-openshift-tests-private-release-4.15__multi-nightly-4.15-upgrade-from-stable-4.14.yaml:- as: gcp-ipi-proxy-private-arm-f14
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: gcp-ipi-proxy-private-amd-f28-destructive
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: gcp-ipi-proxy-private-arm-f14
```

Reply:
ah, since 4.15 is the start version of the upgrade path, so i think 4.15 stable version is expected, right?

Reply:
Actually no, stable ci profiles are also selected from nightly ci, because we want to find issues as early as possible, the stable ci is just to verify signature stuff. so we do not copy each profile from nightly ci to stable ci. And even more, later I will update the stable ci to align stable ci with adm/arm/multi nightly ci, too.  That is on my ToDo list.